tests/ci_build/build_jvm_packages.sh

```diff
@@ -15,7 +15,7 @@ spark_version=$1
 
 rm -rf build/
 cd jvm-packages
-
+export RABIT_MOCK=ON
```

Comment:
Does this script only used to build xgboost-spark to run tests?

Reply:
Yes, it is